webapp/src/components/backstage/template_selector.tsx

```diff
@@ -301,7 +301,118 @@ export const PresetTemplates: PresetTemplate[] = [
+                            'Align on scope, quality, and time.',
+                            'There are likely many different efforts to achieve alignment here, this checkbox just symbolizes sign-off from contributors.',
+                        ),
+                        newChecklistItem('Breakdown feature milestones and add them to this checklist'),
+                    ],
+                },
+                {
+                    title: 'Ship',
+                    items: [
+                        newChecklistItem('Update documentation and user guides'),
+                        newChecklistItem('Merge all feature and bug PRs to master'),
+                        newChecklistItem(
+                            'Demo to the community',
+                            'For example:\n' +
+                            '- R&D meeting\n' +
+                            '- Developer meeting\n' +
+                            '- Company wide meeting\n',
+                        ),
+                        newChecklistItem('Build telemetry dashboard to measure adoption'),
+                        newChecklistItem(
+                            'Create launch kit for go-to-market teams',
+                            'Including but not exclusive to:\n' +
+                            '- release blog post\n' +
+                            '- one-pager\n' +
+                            '- demo video\n',
+                        ),
+                    ],
+                },
+                {
+                    title: 'Follow up',
+                    items: [
+                        newChecklistItem('Schedule meeting to review adoption metrics and user feedback'),
+                        newChecklistItem('Plan improvements and next iteration'),
+                    ],
+                },
+            ],
+            num_actions: 2,
+            create_public_playbook_run: true,
+            message_on_join_enabled: true,
+            message_on_join:
+                'Hello and welcome!\n\n' +
+                'This channel was created as part of the **Feature Lifecycle** playbook and is where conversations related to developing this feature are held. You can customize this message using markdown so that every new channel member can be welcomed with helpful context and resources.',
+            categorize_channel_enabled: true,
+            description:
+                '**One-liner**\n' +
+                '<ie. Enable users to prescribe a description template so it is consistent for every run and therefore easier to read.>\n' +
+                '\n' +
+                '**Targets release**\n' +
+                '- Code complete: date\n' +
+                '- Customer release: month\n' +
+                '\n' +
+                '**Resources**\n' +
+                '- Jira Epic: <link>\n' +
+                '- UX prototype: <link>\n' +
+                '- Technical design: <link>\n' +
+                '- User docs: <link>\n',
```

Comment:
Idea for later: https://www.npmjs.com/package/js-trim-multiline-string

```suggestion
            description: mtrim`
                **One-liner**
                <ie. Enable users to prescribe a description template so it is consistent for every run and therefore easier to read.>
                
                **Targets release**
                - Code complete: date
                - Customer release: month
                
                **Resources**
                - Jira Epic: <link>
                - UX prototype: <link>
                - Technical design: <link>
                - User docs: <link>
            `,
```

Reply:
Ooh, that's neat! Do you know if it preserves /relative/ indentation as needed?

Reply:
I didn't think it did at first, but it does! https://runkit.com/calebroseland/610c0c63be391400136893db (switch to full text output)